python/ray/tests/test_multi_node_2.py

```diff
@@ -113,6 +117,9 @@ def verify_load_metrics(monitor, expected_resource_usage=None, timeout=30):
             raise ValueError("Timeout. {} != {}".format(
                 resource_usage, expected_resource_usage))
 
+    # Sanity check we emitted a resize event.
+    assert any(["Resized to" in x for x in monitor.event_summarizer.summary()])
```

Comment:
The lint test errored because flake8 didn't like some assert statements in `test_autoscaling_policy.py` and  `test_multi_node_2.py`.
I think getting deleting the brackets in the relevant lines `any([...])` -> `any(...)` will fix it.